pkg/solidity-utils/contracts/helpers/CastingHelpers.sol

```diff
@@ -23,7 +23,7 @@ library CastingHelpers {
     /// @dev Returns a native array of addresses as an IERC20[] array. (Used in tests.)
     function asIERC20(address[] memory addresses) internal pure returns (IERC20[] memory tokens) {
         // solhint-disable-next-line no-inline-assembly
-        assembly {
+        assembly ("memory-safe") {
```

Comment:
I just passed through a Stack-too-deep issue today because this function was missing "memory-safe". Cool!